stages/org.osbuild.systemd.unit.create

```diff
@@ -44,6 +48,17 @@ SCHEMA = r"""
         "type": "string",
         "pattern": "^[\\w:.\\\\-]+[@]{0,1}[\\w:.\\\\-]*\\.service$"
     },
+    "unit-type": {
+        "type": "string",
+        "enum": ["system", "global"],
+        "default": "system",
+        "description": "Selects between systemd system or global unit to add dropin"
+    },
+    "persistent": {
+        "type": "boolean",
+        "default": false,
+        "description": "Define if systemd unit file should survive upgrade"
```

Comment:
Survive what upgrade? Perhaps I haven't been keeping track of all these changes too much.

Reply:
I meant `rpm-ostree upgrade` , will explicitly mention that.

Reply:
added.